web-registry/src/components/organisms/MediaForm/MediaForm.tsx

```diff
@@ -0,0 +1,149 @@
+type MediaErrors = MediaErrorsSimple | MediaErrorsLegacy;
+
+export function isSimpleValues(
+  _values: MediaValues,
+  ccId?: Maybe<string> | string,
+): _values is MediaValuesSimple {
+  return !!ccId;
+}
+
+function isSimpleErrors(
+  _errors: MediaErrors,
+  ccId?: Maybe<string> | string,
+): _errors is MediaErrorsSimple {
+  return !!ccId;
+}
+
+export const cropAspect = { aspect: 322 / 211 }; // px values pulled from mockups (width / height)
+
+interface MediaFormProps {
+  submit: (values: MediaValues) => Promise<void>;
+  onPrev?: () => void;
+  onNext?: () => void;
+  initialValues: MediaValues;
+  creditClassId?: string | null;
+  graphData?: ShaclGraphByUriQuery;
+}
+
+/** Formik Context + handlers for legacy and new media */
+export const MediaFormContext = ({
+  initialValues,
+  graphData,
+  creditClassId,
+  ...props
+}: MediaFormProps): JSX.Element => {
+  const { confirmSave, isEdit } = useProjectEditContext();
+
+  async function handleSubmit(
+    values: MediaValues,
+    { setSubmitting, setTouched }: FormikHelpers<MediaValues>,
+  ): Promise<void> {
+    setSubmitting(true);
+    try {
+      await props.submit(values);
+      setSubmitting(false);
+      setTouched({}); // reset to untouched
+      if (isEdit && confirmSave) confirmSave();
+    } catch (e) {
+      setSubmitting(false);
+    }
+  }
+
+  const handleValidate = async (values: MediaValues): Promise<MediaErrors> => {
+    const errors: MediaErrors = {};
+    if (graphData?.shaclGraphByUri?.graph) {
+      const projectPageData = {
+        ...getProjectPageBaseData(creditClassId),
+        ...values,
+      };
+      const report = await validate(
+        graphData.shaclGraphByUri.graph,
+        projectPageData,
+        'http://regen.network/ProjectPageMediaGroup',
+      );
+      for (const result of report.results) {
+        const path: string = result.path.value;
+        const compactedPath = getCompactedPath(path) as
+          | keyof MediaValues
+          | undefined;
+        if (compactedPath) {
+          if (
+            [
+              'regen:previewPhoto',
+              'regen:videoURL',
+              'regen:landStewardPhoto',
+            ].includes(compactedPath)
+          ) {
+            errors[compactedPath] = { '@value': requiredMessage };
+          }
+          if (compactedPath === 'regen:galleryPhotos') {
+            if (!isSimpleErrors(errors, creditClassId)) {
+              errors[compactedPath] = 'You must add 4 photos';
+            }
+          }
+        }
+      }
+    }
+    return errors;
+  };
+
+  return (
+    <OnBoardingCard>
+      <Formik
+        enableReinitialize
+        validateOnMount
+        initialValues={initialValues}
+        validate={handleValidate}
+        onSubmit={handleSubmit}
+      >
+        {({ submitForm, isValid, isSubmitting, touched }) => (
+          <>
+            {!!creditClassId ? <MediaFormSimple /> : <MediaFormLegacy />}
```

Comment:
Based on the convo this morning, I'm not sure this is correct - do we want to display the simple form regardless? Are there any situations we should still render the legacy form?